patches/react-native-vision-camera+2.16.2.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt
+index c0a8b23..653b51e 100644
+--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt
++++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt
+@@ -40,7 +40,7 @@ class FrameProcessorRuntimeManager(context: ReactApplicationContext, frameProces
+       val holder = context.catalystInstance.jsCallInvokerHolder as CallInvokerHolderImpl
+       mScheduler = VisionCameraScheduler(frameProcessorThread)
+       mContext = WeakReference(context)
+-      mHybridData = initHybrid(context.javaScriptContextHolder.get(), holder, mScheduler!!)
++      mHybridData = initHybrid(context.javaScriptContextHolder!!.get(), holder, mScheduler!!)
```

Comment:
Curious if this is fixed in latest version 3.6.12

Reply:
I think you mean version `2.16.5` since we would not want to bump it to different major 😅 ? And it is not fixed there unfortunately: https://github.com/mrousavy/react-native-vision-camera/blob/v2.16.5/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt#L43

Reply:
We are upgrading to vision camera 3 in https://github.com/Expensify/App/issues/30732

However, is there an issue or upstream PR for this? We work closely with Margelo and can definitely ask them to review it

Reply:
cc @mrousavy

Reply:
@mrousavy Do you have an issue to track this fix?

Reply:
No issue to track this, in V3 it is already fixed with a proper error message if the app is running in bridge-less mode (that's when this property is null).

I also just backported this fix to V2, for now you can roll this patch just fine, in the next version of VisionCamera V2 (or V3) you will no longer need this 👍